src/FhirClientContext.ts

```diff
@@ -7,12 +7,14 @@ export type FhirClientContextType = {
     client: Client;
     patient: Patient;
     carePlan: CarePlan;
```

Comment:
Could you rename this to currentCarePlan so it’s more obvious what the difference is between careplan and allcareplans?

Reply:
thanks @hannahburkhardt  I will rename it.   Sorry,  I didn’t have time as I believe this is used in quite a few places.